### PR TITLE
Display gzip size of built assets

### LINKED ISSUE #57: Display gzip size with Assets output on build
Would be nice to also show gzip size along with the assets output:

```
641.23 KB    (178  KB gzip)   build/client/main-618bbd3a04be91bfc336.js
164.22 KB    (23.5 KB gzip)   build/client/main-3e90752ad5a140255589.css
```

Can use [gzip-size](https://github.com/sindresorhus/gzip-size)

### PR DESCRIPTION
**Pull request by dting**
Resolves #57

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "file-loader": "0.9.0",
     "filesize": "3.3.0",
     "glob": "7.1.0",
+    "gzip-size": "^3.0.0",
```
Comment (tizmagik) [inline package.json]: Can we pin this at 3.0.0?

Comment (dting) [inline package.json]: Oops. Sounds good.

File: utils/printAssets.js [hunk 2]
```diff
@@ -1,33 +1,52 @@
...
 // Prints webpack asset stats
+const fs = require('fs');
 const path = require('path');
 const filesize = require('filesize');
+const gzipSize = require('gzip-size');
 const stripAnsi = require('strip-ansi');
 const logger = require('../cli/logger');
 
-module.exports = (stats) => {
+module.exports = (stats, clientConfig) => {
```
Comment (ccpricenytimes) [inline utils/printAssets.js]: Hey @dting Make sure to pass clientConfig in here https://github.com/NYTimes/kyt/blob/master/cli/actions/build.js#L51

**Comment by ccpricenytimes** [comment]
thanks @dting !

**Pull request merged**